.github/workflows/sub_lint.yml

```diff
@@ -77,7 +77,73 @@ on:
         type: boolean
         required: false
       cpplintFilters:
-        default: -whitespace-*
+        default: >-
+          '
+          +build/deprecated,
+          +build/endif_comment,
+          +build/explicit_make_pair,
+          +build/forward_decl,
+          +build/header_guard,
+          +build/include,
+          +build/include_subdir,
+          +build/include_alpha,
+          -build/include_order,
+          +build/include_what_you_use,
+          +build/namespaces_headers,
+          +build/namespaces_literals,
+          +build/namespaces,
+          +build/printf_format,
+          +build/storage_class,
+          +readability/alt_tokens,
+          -readability/braces,
```

Comment:
From what I have seen this looks like a sensible rule or do I miss something?

Reply:
It is sensible, but IMO all formatting issues should be handled by clang-format -- rather than issuing a linter warning (which is just noise when old non-conforming code is touched, we have not been asking developers to update that code to current standards), our pre-commit hook should be fixing this automatically. We are not yet 100% clang-formatted, but IMO we should be working to get there, at which point this warning is obsolete.